module/core/src/Application/Request/ParamConverter/AggregateRootParamConverter.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Core\Application\Request\ParamConverter;
+
+use Ergonode\EventSourcing\Domain\AbstractAggregateRoot;
+use Ergonode\EventSourcing\Infrastructure\Manager\EventStoreManager;
+use Ergonode\SharedKernel\Domain\AggregateId;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Request\ParamConverter\ParamConverterInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Webmozart\Assert\Assert;
+
+class AggregateRootParamConverter implements ParamConverterInterface
+{
+    private EventStoreManager $manager;
+
+    public function __construct(EventStoreManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    public function apply(Request $request, ParamConverter $configuration): void
+    {
+        $name = $configuration->getName();
+
+        $parameter = $request->get($name);
```

Comment:
It is possilbe that we had situation like parameter name is not the same as name of param converter?

Reply:
No, that is why I remove name from actions